server/routes/accessibilityStatement/index.ts

```diff
@@ -0,0 +1,14 @@
+import type { Router } from 'express'
+import { checkUserHasViewAuthority } from '../../middleware/roleBasedAccessControl'
+import AccessibilityStatementController from './accessibilityStatementController'
+
+/**
+ * Route definition for the accessibility statement page
+ */
+export default (router: Router) => {
+  const accessibilityStatementController = new AccessibilityStatementController()
+  router.get('/accessibility-statement', [
+    checkUserHasViewAuthority(),
```

Comment:
No, we want it to be fully open access. The check you've used here is for view access/role for PLP data (ie. read only access to someone's plan/goals etc).
In the case of this page we want anyone to be able to access it, so you'll need to remove this method. It _might_ also need to be fully open, in that you don't even need to be authenticated to see it? Perhaps check how other services have set their pages up in this respect (I would suggest Send Legal Mail, because I know about that project and can answer any questions/context you might have)

I would also be tempted not to use a controller at all - we really dont need one in this case. You can write the handler function inline as an anonymous function, like this:

```
router.get('/accessibility-statement', async (req, res, next) => {
    res.render('pages/accessibilityStatement/index')
})
```
Which means you can delete the controller class altogether

Reply:
And we'll want a single simple cypress test to assert a user without any of our roles can access the page